package.json

```diff
@@ -10,7 +10,7 @@
     "ci:lint-docs": "yarn generate docs && test -z \"$(git status --porcelain)\" || echo 'The root README has not been updated. Run `yarn generate docs` in the root of your quilt directory and try again.'",
     "test": "jest --maxWorkers=3 --coverage=false",
     "check": "lerna run check",
-    "release": "lerna publish && git push --follow-tags",
+    "release": "lerna publish",
```

Comment:
🙏love it. I personally like to do a quick scan locally before pushing to master. This makes that possible now